cosinnus/static/less/variables.less

```diff
@@ -111,19 +111,19 @@
 @map-followed-color: #00d000;
 
 @map-button-color: #FFF;
-@map-action-button-color: #34b4b5;
+@map-action-button-color: #268282;
```

Comment:
![new_map_buttons](https://github.com/user-attachments/assets/68cc24a4-cd38-4791-8a5a-b01a6fa586ce)

Ich habe die Farben verdunkelt, bis sie einen ausreichenden Kontrast hatten und etwas getweakt. Wenn man Geld verdunkelt, wird es grünbraun, ich habe also eine neue Farbe gewählt. 

Bei der Actionfarbe. z.B. "Clear Filters"-Button, kam beim Verdunkeln eine Farbe raus, die sehr nach wechange aussieht. Eventuell suche ich hier noch was anderes raus. 

Bei den Topics habe ich schwarz als Hintergrund gewählt. In den neuen Apps (zB Deck und Kalendar) ist das die "neutrale" Actionfarbe.